components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -172,12 +336,21 @@ public List<Property> getConfigurationProperties() {
 
         List<Property> configProperties = new ArrayList<Property>();
 
+        Property googleOneTap = new Property();
+        googleOneTap.setName(OIDCAuthenticatorConstants.GOOGLE_ONE_TAP_ENABLED);
+        googleOneTap.setDisplayName("Google One Tap");
+        googleOneTap.setRequired(false);
+        googleOneTap.setType("boolean");
+        googleOneTap.setDescription("Enable Google One Tap as a sign in option.");
+        googleOneTap.setDisplayOrder(1);
+        configProperties.add(googleOneTap);
+
         Property clientId = new Property();
         clientId.setName(OIDCAuthenticatorConstants.CLIENT_ID);
         clientId.setDisplayName("Client ID");
         clientId.setRequired(true);
         clientId.setDescription("The client identifier value of the Google identity provider.");
-        clientId.setDisplayOrder(1);
+        clientId.setDisplayOrder(2);
```

Comment:
@JKAUSHALYA @brionmario is this ok to change the existing order? Editing the order will display the configs in a much more organized way. But this is affecting only the carbon console and this is not a really big concern. Here my only worry is whether we are breaking the backward compatibility behavior. WDYT?

Reply:
This is just the display order right. IMO, it shouldn't affect the functionality.